Sources/StreamChatUI/ChatMessageList/Attachments/ChatMessageImageGallery+ImagePreview.swift

```diff
@@ -69,17 +69,12 @@ extension _ChatMessageImageGallery {
         override open func updateContent() {
             let attachment = content
 
-            if let url = attachment?.payload?.imagePreviewURL {
-                loadingIndicator.isVisible = true
-                imageTask = loadImage(with: url, options: .shared, into: imageView, completion: { [weak self] _ in
+            loadingIndicator.isVisible = true
+            imageTask = imageView
+                .loadImage(from: attachment?.payload?.imagePreviewURL, resizeAutomatically: false) { [weak self] _ in
```

Comment:
Why `resizeAutomatically: false` here? This is just the preview, it should be resized imo.

Reply:
the `imageView` does not have appropriate size but calling `imageView.layoutIfNeeded` fixes that